src/common/utils/playerDynamics.ts

```diff
@@ -54,46 +76,24 @@ export const getControlsMovementVector = (
     Number(params.isMovingBackward) - Number(params.isMovingForward),
   ]);
 
-  // Get the direction from the keyboard, apply the yaw, and normalize to get a unit vector
-  target
+  // Gets the direction from the keyboard, apply the yaw, and normalize to get a unit vector
+  inputVelocity
     .addVectors(GlobalControls.xInputVelocity, GlobalControls.zInputVelocity)
     .applyQuaternion(GlobalControls.yaw)
-    .normalize();
+    .setLength(PlayerConstants.movementSpeed);
 };
 
-type PlayerCollisionParams = {
-  playerCurrentPosition: Vector3;
-  playerRadius: number;
-  playerRay: Line3;
-  playerWorldMatrix: Matrix4;
-  boundsTree?: MeshBVH;
+type CapsuleBVHParams = {
+  capsule: Mesh;
+  bvh?: MeshBVH;
```

Comment:
Would prefer explicitness -- dislike abbreviations and it's easier to understand what's going on when it's clear what BVH is

```suggestion
 boundingVolumeHierarchy
?: MeshBVH;
```

Reply:
I generally agree, but bvh is a pretty standard industry term, and a widely known data structure in the graphics world as well. Plus there's a notion doc explaining its behavior -- i prefer the abbreviation. If you feel strongly I can change it to the lengthier alternative

Reply:
Let's keep `bvh` then -- if this becomes a common point of confusion in the code we'll change later